### PR TITLE
{bio}[system] AptaSUITE v0.9.4 + JavaFX v11.0.2

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

File: easybuild/easyconfigs/a/AptaSUITE/AptaSUITE-0.9.4-Java-11.eb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    ('Java', '11'),
+    ('JavaFX', '11.0.2', '_linux-x64_bin-sdk'),
+]
+
+postinstallcmds = ["cd %(installdir)s && ln -s aptasuite-%(version)s.jar aptasuite.jar"]
+
+modaliases = {
+    'aptasuite-gui': 'java -Dprism.order=sw --module-path ${EBROOTJAVAFX}/lib ' +
+                     '--add-modules javafx.controls,javafx.fxml -jar ${EBROOTAPTASUITE}/aptasuite.jar',
+                     # split in two lines to make travis happy (line length below 120 characters)
```
Comment (smoors) [inline easybuild/easyconfigs/a/AptaSUITE/AptaSUITE-0.9.4-Java-11.eb]: please remove this comment?

Comment (smoors) [reply easybuild/easyconfigs/a/AptaSUITE/AptaSUITE-0.9.4-Java-11.eb]: why not also add this:
```
'aptasuite': 'java -jar ${EBROOTAPTASUITE}/aptasuite.jar',
```

**Review by smoors — DISMISSED** [review]

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3100.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/00e079893dced1a6238c466c5f5910a9 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @pescobar!

**Pull request merged**